AWSS3/AWSS3TransferUtility.m

```diff
@@ -762,6 +776,30 @@ - (void)URLSession:(NSURLSession *)session
         
         NSHTTPURLResponse *HTTPResponse = (NSHTTPURLResponse *)task.response;
         NSMutableDictionary *userInfo = [NSMutableDictionary dictionaryWithDictionary:[HTTPResponse allHeaderFields]];
+
+        NSData *responseData = self.responseData[@(task.taskIdentifier)];
+        [self.responseData removeObjectForKey:@(task.taskIdentifier)];
+      
+        if (!responseData) {
+          if ([task isKindOfClass:[NSURLSessionDownloadTask class]]) {
+            AWSS3TransferUtilityDownloadTask *downloadTask = [self getDownloadTask:(NSURLSessionDownloadTask *)task];
+            responseData = downloadTask.data;
+          }
+        }
+      
+        NSString *responseString = [[NSString alloc] initWithData: responseData encoding:NSUTF8StringEncoding];
+        if ([responseString rangeOfString:@"<Error>"].location != NSNotFound) {
+          AWSXMLDictionaryParser *xmlParser = [AWSXMLDictionaryParser new];
+          xmlParser.trimWhiteSpace = YES;
+          xmlParser.stripEmptyNodes = NO;
+          xmlParser.wrapRootNode = YES; //wrapRootNode for easy process
+          xmlParser.nodeNameMode = AWSXMLDictionaryNodeNameModeNever; //do not need rootName anymore since rootNode is wrapped.
+          
+          NSDictionary *responseDict = [xmlParser dictionaryWithString:responseString];
+          userInfo[@"Error"] = responseDict[@"Error"];
+          AWSDDLogDebug(@"Error response received from S3: %@", responseDict);
```

Comment:
Can you please change this to `AWSDDLogError`

Reply:
Done